### PR TITLE
Page/thankyouforapplying

### LINKED ISSUE #10: Thank you for applying page
- Build the following page designed in the figma

### PR DESCRIPTION
**Pull request by jaredsbt**
## Tracking Info

Resolves #10

## Changes

* Created ThankyouForApplying component and css module under components

## Testing

* Made the ThankyouForApplying as index element for easy access and viewing (will change back to candidates after pr approval)
* Figma comparison 
* "here" links to candidates page
* Return to Main page links to CCIDC Home Page

## Confirmation of Change

linkToApplicationPage
linkToHomePage
ThankYouForApplying

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ function App() {
     <div className="app-container">
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route index element={<Candidates />} />
+          <Route index element={<ThankyouForApplying />} />
```
Comment (Miyuki-L) [inline frontend/src/App.tsx:20]: Keep the index element as Candidates
and then add a new route path for the thankyou page.

File: frontend/src/components/ThankyouForApplying.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import styles from "./ThankyouForApplying.module.css";
+
+import { Page } from "../components/index.ts";
```
Comment (Miyuki-L) [inline frontend/src/components/ThankyouForApplying.tsx:3]: address this. It should be fixed using `npm run lint-fix` in the frontend directory

File: frontend/src/components/ThankyouForApplying.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          href="https://ccidc.org/becoming-a-certified-interior-designer/"
+          className={styles.lightBlue}
+        >
+          here
+        </a>{" "}
+        and reapply when you have met the requirements for at least path one.{" "}
+      </p>
+
+      <div className={styles.centeredContainer}>
+        <a href="https://ccidc.org/">
```
Comment (Miyuki-L) [inline frontend/src/components/ThankyouForApplying.tsx:23]: Lets use the button componenet. Pull from main and use the new "additionalStyles" tag to apply the necessary styles.

For the page rounting you can look at Hrithik's [CongratulationsPopup](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/main/frontend/src/components/CongratulationsPopupWindow.tsx#L63)

File: frontend/src/pages/index.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Apply } from "./Apply.tsx";
 import { Candidates } from "./Candidates.tsx";
```
Comment (Miyuki-L) [inline frontend/src/pages/index.ts:2]: Same for these warnings a run of `npm run lint-fix` in the approprate directory should work.

**Review by Miyuki-L — APPROVED** [review]
Looks pretty good. Just address those comments and pull from main.

File: frontend/src/components/ThankyouForApplying.tsx
Comment (Miyuki-L) [inline frontend/src/components/ThankyouForApplying.tsx:1]: Move this to same folder as pages. This is a more a page than a component

**Review by hpai1 — APPROVED** [review]
Looks good to me other than Helen's comments.

**Pull request merged**